### DISCUSSION TITLE
Hosting react emails

### DISCUSSION THREAD

**Discussion by shahriyardx** [resend/react-email#998] [Q&A]
I have developed some templates of react emails on my existing next 13 project. Now how can I host it, also the question is, should I host it on the internet or not? 

My Project structure
```
.
└── root/
    ├── email/
    │   ├── emails/
    │   │   └── something.tsx
    │   ├── package.json
    │   └── yarn.lock
    ├── src/
    │   ├── app
    │   └── components
    ├── package.json
    └── yarn.lock
```

**Comment by gabrielmfern** [comment]
What do you mean by hosting? Do you mean storing the source code somewhere like github or keeping the rendered emails somewhere?

**Reply by shahriyardx** [reply]
Just like we make our websites live on some domain https://something.com

From what I understand that this website does not need to be hosted anywhere

Just import the component and send and templates are static and dynamic with props

**Reply by jinsley8** [reply]
@shahriyardx 

You don't need that `package.json` in the `/email` folder.

The current `react-email` automatic setup is confusing. `react-email` only needs to be a devDependency and add `.react-email` to `.gitignore`.

Put `/emails/` in the root folder. The app it creates when you run `yarn email dev` is inside `.react-email` and only needed in local development.

```
// package.json in root
"dependencies": {
    "@react-email/components": "0.0.10",
    "@react-email/render": "0.0.8",
},
"devDependencies": {
    "react-email": "1.9.5",
},
```

```
// .gitignore
.react-email
```
